### ISSUE TITLE
The pre-trained model can't achieve the accuracy presented in the paper

### ISSUE THREAD

**Issue by windpls** [hwwang55/GraphGAN#7]
Hi hwwang55, I cloned the repo and wrote the code below to evaluate the performance of pre-trained model on CA-GrQc test set, but only got 75.98% using generator embedding, which is far away from 84.9% stated in the paper. Is there anything wrong?

```
n_node = 5242
def evaluation():
    lpe = lp.LinkPredictEval(
        config.pretrain_emb_filename_g,
        config.test_filename,
        config.test_neg_filename,
        n_node,
        config.n_emb
    )
    result = lpe.eval_link_prediction()
    print(result)

if __name__ == '__main__':
    evaluation()
```

**Comment by hwwang55** [comment]
Did you try to increase the value of update_ratio in config.py? And, you can also try tuning other hyperparameters.

**Comment by windpls** [comment]
@hwwang55 No, I didn't tune the model further and just loaded the embedding in "pre_train/link_prediction/CA-GrQc_pre_train.emb". Is it necessary to fine tune the model in order to get 84%+ accuracy?

**Comment by windpls** [comment]
I think I
